### ISSUE TITLE
Convert ONNX to TensorRT engine fail

### ISSUE THREAD

**Issue by yueyihua** [NVIDIA/TensorRT#1127]
## Description

Convert my onnx model to tensorrt engine fail：
$ gdb --args trtexec --onnx=stable_hopenetlite.onnx --saveEngine=stable_hopenetlite.trt --minShapes=input:1x3x224x224 --optShapes=input:16x3x224x224 --maxShapes=input:16x3x224x224
GNU gdb (Ubuntu 8.1.1-0ubuntu1) 8.1.1
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "aarch64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
<http://www.gnu.org/software/gdb/documentation/>.
For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from trtexec...(no debugging symbols found)...done.
(gdb) r
Starting program: /usr/src/tensorrt/bin/trtexec --onnx=stable_hopenetlite.onnx --saveEngine=stable_hopenetlite.trt --minShapes=input:1x3x224x224 --optShapes=input:16x3x224x224 --maxShapes=input:16x3x224x224
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/aarch64-linux-gnu/libthread_db.so.1".
&&&& RUNNING TensorRT.trtexec # /usr/src/tensorrt/bin/trtexec --onnx=stable_hopenetlite.onnx --saveEngine=stable_hopenetlite.trt --minShapes=input:1x3x224x224 --optShapes=input:16x3x224x224 --maxShapes=input:16x3x224x224
[03/17/2021-15:05:01] [I] === Model Options ===
[03/17/2021-15:05:01] [I] Format: ONNX
[03/17/2021-15:05:01] [I] Model: stable_hopenetlite.onnx
[03/17/2021-15:05:01] [I] Output:
[03/17/2021-15:05:01] [I] === Build Options ===
[03/17/2021-15:05:01] [I] Max batch: explicit
[03/17/2021-15:05:01] [I] Workspace: 16 MB
[03/17/2021-15:05:01] [I] minTiming: 1
[03/17/2021-15:05:01] [I] avgTiming: 8
[03/17/2021-15:05:01] [I] Precision: FP32
[03/17/2021-15:05:01] [I] Calibration:
[03/17/2021-15:05:01] [I] Safe mode: Disabled
[03/17/2021-15:05:01] [I] Save engine: stable_hopenetlite.trt
[03/17/2021-15:05:01] [I] Load engine:
[03/17/2021-15:05:01] [I] Builder Cache: Enabled
[03/17/2021-15:05:01] [I] NVTX verbosity: 0
[03/17/2021-15:05:01] [I] Inputs format: fp32:CHW
[03/17/2021-15:05:01] [I] Outputs format: fp32:CHW
[03/17/2021-15:05:01] [I] Input build shape: input=1x3x224x224+16x3x224x224+16x3x224x224
[03/17/2021-15:05:01] [I] Input calibration shapes: model
[03/17/2021-15:05:01] [I] === System Options ===
[03/17/2021-15:05:01] [I] Device: 0
[03/17/2021-15:05:01] [I] DLACore:
[03/17/2021-15:05:01] [I] Plugins:
[03/17/2021-15:05:01] [I] === Inference Options ===
[03/17/2021-15:05:01] [I] Batch: Explicit
[03/17/2021-15:05:01] [I] Input inference shape: input=16x3x224x224
[03/17/2021-15:05:01] [I] Iterations: 10
[03/17/2021-15:05:01] [I] Duration: 3s (+ 200ms warm up)
[03/17/2021-15:05:01] [I] Sleep time: 0ms
[03/17/2021-15:05:01] [I] Streams: 1
[03/17/2021-15:05:01] [I] ExposeDMA: Disabled
[03/17/2021-15:05:01] [I] Spin-wait: Disabled
[03/17/2021-15:05:01] [I] Multithreading: Disabled
[03/17/2021-15:05:01] [I] CUDA Graph: Disabled
[03/17/2021-15:05:01] [I] Skip inference: Disabled
[03/17/2021-15:05:01] [I] Inputs:
[03/17/2021-15:05:01] [I] === Reporting Options ===
[03/17/2021-15:05:01] [I] Verbose: Disabled
[03/17/2021-15:05:01] [I] Averages: 10 inferences
[03/17/2021-15:05:01] [I] Percentile: 99
[03/17/2021-15:05:01] [I] Dump output: Disabled
[03/17/2021-15:05:01] [I] Profile: Disabled
[03/17/2021-15:05:01] [I] Export timing to JSON file:
[03/17/2021-15:05:01] [I] Export output to JSON file:
[03/17/2021-15:05:01] [I] Export profile to JSON file:
[03/17/2021-15:05:01] [I]
[New Thread 0x7f99e739b0 (LWP 23974)]
----------------------------------------------------------------
Input filename:   stable_hopenetlite.onnx
ONNX IR version:  0.0.6
Opset version:    10
Producer name:    pytorch
Producer version: 1.7
Domain:
Model version:    0
Doc string:
----------------------------------------------------------------
[03/17/2021-15:05:04] [W] [TRT] onnx2trt_utils.cpp:220: Your ONNX model has been generated with INT64 weights, while TensorRT does not natively support INT64. Attempting to cast down to INT32.
[03/17/2021-15:05:04] [W] [TRT] DLA requests all profiles have same min, max, and opt value. All dla layers are falling back to GPU
[03/17/2021-15:05:04] [I] [TRT]
[03/17/2021-15:05:04] [I] [TRT] --------------- Layers running on DLA:
[03/17/2021-15:05:04] [I] [TRT]
[03/17/2021-15:05:04] [I] [TRT] --------------- Layers running on GPU:
[03/17/2021-15:05:04] [I] [TRT] Conv_0 + Relu_1, MaxPool_2, Conv_6 + Relu_7, Conv_3, Conv_4 + Relu_5, Conv_8, Conv_9 + Relu_10, Reshape_13 + Transpose_14, Reshape_16, Split_17_1, Conv_18 + Relu_19, Conv_20, Conv_21 + Relu_22, Split_17, Reshape_25 + Transpose_26, Reshape_28, Split_29_1, Conv_30 + Relu_31, Conv_32, Conv_33 + Relu_34, Split_29, Reshape_37 + Transpose_38, Reshape_40, Split_41_1, Conv_42 + Relu_43, Conv_44, Conv_45 + Relu_46, Split_41, Reshape_49 + Transpose_50, Reshape_52, Conv_56 + Relu_57, Conv_53, Conv_54 + Relu_55, Conv_58, Conv_59 + Relu_60, Reshape_63 + Transpose_64, Reshape_66, Split_67_1, Conv_68 + Relu_69, Conv_70, Conv_71 + Relu_72, Split_67, Reshape_75 + Transpose_76, Reshape_78, Split_79_1, Conv_80 + Relu_81, Conv_82, Conv_83 + Relu_84, Split_79, Reshape_87 + Transpose_88, Reshape_90, Split_91_1, Conv_92 + Relu_93, Conv_94, Conv_95 + Relu_96, Split_91, Reshape_99 + Transpose_100, Reshape_102, Split_103_1, Conv_104 + Relu_105, Conv_106, Conv_107 + Relu_108, Split_103, Reshape_111 + Transpose_112, Reshape_114, Split_115_1, Conv_116 + Relu_117, Conv_118, Conv_119 + Relu_120, Split_115, Reshape_123 + Transpose_124, Reshape_126, Split_127_1, Conv_128 + Relu_129, Conv_130, Conv_131 + Relu_132, Split_127, Reshape_135 + Transpose_136, Reshape_138, Split_139_1, Conv_140 + Relu_141, Conv_142, Conv_143 + Relu_144, Split_139, Reshape_147 + Transpose_148, Reshape_150, Conv_154 + Relu_155, Conv_151, Conv_152 + Relu_153, Conv_156, Conv_157 + Relu_158, Reshape_161 + Transpose_162, Reshape_164, Split_165_1, Conv_166 + Relu_167, Conv_168, Conv_169 + Relu_170, Split_165, Reshape_173 + Transpose_174, Reshape_176, Split_177_1, Conv_178 + Relu_179, Conv_180, Conv_181 + Relu_182, Split_177, Reshape_185 + Transpose_186, Reshape_188, Split_189_1, Conv_190 + Relu_191, Conv_192, Conv_193 + Relu_194, Split_189, Reshape_197 + Transpose_198, Reshape_200, Conv_201 + Relu_202, ReduceMean_203, fc_y.weight, fc_p.weight, fc_r.weight, Gemm_206, Gemm_205, Gemm_204, (Unnamed Layer* 187) [Constant] + (Unnamed Layer* 188) [Shuffle], (Unnamed Layer* 192) [Constant] + (Unnamed Layer* 193) [Shuffle], (Unnamed Layer* 197) [Constant] + (Unnamed Layer* 198) [Shuffle], (Unnamed Layer* 199) [ElementWise], (Unnamed Layer* 194) [ElementWise], (Unnamed Layer* 189) [ElementWise],
[New Thread 0x7f91f229b0 (LWP 23975)]
[03/17/2021-15:05:11] [I] [TRT] Some tactics do not have sufficient workspace memory to run. Increasing workspace size may increase performance, please check verbose output.
[03/17/2021-15:05:16] [E] [TRT] ../builder/cudnnBuilderUtils.cpp (427) - Cuda Error in findFastestTactic: 700 (an illegal memory access was encountered)
[03/17/2021-15:05:16] [E] [TRT] ../rtSafe/safeRuntime.cpp (32) - Cuda Error in free: 700 (an illegal memory access was encountered)
terminate called after throwing an instance of 'nvinfer1::CudaError'
  what():  std::exception

Thread 1 "trtexec" received signal SIGABRT, Aborted.
__GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
51      ../sysdeps/unix/sysv/linux/raise.c: No such file or directory.
(gdb) bt
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x0000007fa31178d4 in __GI_abort () at abort.c:79
#2  0x0000007fa33ad10c in __gnu_cxx::__verbose_terminate_handler() () from /usr/lib/aarch64-linux-gnu/libstdc++.so.6
#3  0x0000007fa33aac54 in ?? () from /usr/lib/aarch64-linux-gnu/libstdc++.so.6
#4  0x0000007fa33a9b5c in ?? () from /usr/lib/aarch64-linux-gnu/libstdc++.so.6
#5  0x0000007fa33aa340 in __gxx_personality_v0 () from /usr/lib/aarch64-linux-gnu/libstdc++.so.6
#6  0x0000007fa324a9b4 in ?? () from /lib/aarch64-linux-gnu/libgcc_s.so.1
#7  0x0000007fa324af6c in _Unwind_Resume () from /lib/aarch64-linux-gnu/libgcc_s.so.1
#8  0x0000007fab1418d0 in nvinfer1::throwCudaError(char const*, char const*, int, int, char const*) () from /usr/lib/aarch64-linux-gnu/libnvinfer.so.7
#9  0x0000007fab1253bc in nvinfer1::internal::DefaultAllocator::free(void*) () from /usr/lib/aarch64-linux-gnu/libnvinfer.so.7
#10 0x0000007fab13d728 in nvinfer1::trtCudaFree(nvinfer1::IGpuAllocator*, void*, char const*, char const*, int) () from /usr/lib/aarch64-linux-gnu/libnvinfer.so.7
#11 0x0000007fab0b07a0 in nvinfer1::builder::EngineTacticSupply::LocalBlockAllocator::~LocalBlockAllocator() () from /usr/lib/aarch64-linux-gnu/libnvinfer.so.7
#12 0x0000007fab0a3cd0 in nvinfer1::builder::EngineTacticSupply::getBestTactic(nvinfer1::builder::Node&, nvinfer1::query::Ports<nvinfer1::builder::SymbolicFormat> const&, bool, nvinfer1::builder::AutoDeletingVector<nvinfer1::builder::Algorithm>*) () from /usr/lib/aarch64-linux-gnu/libnvinfer.so.7
#13 0x0000007faafe3e48 in ?? () from /usr/lib/aarch64-linux-gnu/libnvinfer.so.7
#14 0x0000007faafdf91c in nvinfer1::builder::chooseFormatsAndTactics(nvinfer1::builder::Graph&, nvinfer1::builder::TacticSupply&, std::unordered_map<std::string, std::vector<nvinfer1::builder::DynamicRangeSymbol, std::allocator<nvinfer1::builder::DynamicRangeSymbol> >, std::hash<std::string>, std::equal_to<std::string>, std::allocator<std::pair<std::string const, std::vector<nvinfer1::builder::DynamicRangeSymbol, std::allocator<nvinfer1::builder::DynamicRangeSymbol> > > > >*, nvinfer1::NetworkBuildConfig const&) ()
   from /usr/lib/aarch64-linux-gnu/libnvinfer.so.7
#15 0x0000007fab0a8a04 in ?? () from /usr/lib/aarch64-linux-gnu/libnvinfer.so.7
#16 0x0000007fab0ae0e4 in nvinfer1::builder::buildEngine(nvinfer1::NetworkBuildConfig&, nvinfer1::NetworkQuantizationConfig const&, nvinfer1::builder::EngineBuildContext const&, nvinfer1::Network const&) () from /usr/lib/aarch64-linux-gnu/libnvinfer.so.7
#17 0x0000007fab0c4a50 in nvinfer1::builder::Builder::buildInternal(nvinfer1::NetworkBuildConfig&, nvinfer1::NetworkQuantizationConfig const&, nvinfer1::builder::EngineBuildContext const&, nvinfer1::Network const&) () from /usr/lib/aarch64-linux-gnu/libnvinfer.so.7
#18 0x0000007fab0c5a48 in nvinfer1::builder::Builder::buildEngineWithConfig(nvinfer1::INetworkDefinition&, nvinfer1::IBuilderConfig&) ()
   from /usr/lib/aarch64-linux-gnu/libnvinfer.so.7
#19 0x0000005555580964 in sample::networkToEngine(sample::BuildOptions const&, sample::SystemOptions const&, nvinfer1::IBuilder&, nvinfer1::INetworkDefinition&, std::ostream&) ()
#20 0x0000005555581e48 in sample::modelToEngine(sample::ModelOptions const&, sample::BuildOptions const&, sample::SystemOptions const&, std::ostream&) ()
#21 0x0000005555582124 in sample::getEngine(sample::ModelOptions const&, sample::BuildOptions const&, sample::SystemOptions const&, std::ostream&) ()
#22 0x000000555555b3ec in main ()
(gdb) q

## Environment

**TensorRT Version**:   7.1.3.0
**NVIDIA GPU**:           Jetson xavier nx
**NVIDIA Driver Version**: 
**CUDA Version**:        10.2
**CUDNN Version**:     8.0.0.180
**Operating System**:  Ubuntu 18.04
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: 

## Relevant Files
The pytorch model url：https://github.com/OverEuro/deep-head-pose-lite
My python convert code：
`import torch
import stable_hopenetlite
pos_net = stable_hopenetlite.shufflenet_v2_x1_0()
saved_state_dict = torch.load('model/shuff_epoch_120.pkl', map_location="cpu")
pos_net.load_state_dict(saved_state_dict, strict=False)
pos_net.eval()

# Input to the model
batch_size = 1
x = torch.randn(batch_size, 3, 224, 224, requires_grad=False)
torch_out = pos_net(x)

# Export the model
torch.onnx.export(pos_net,               # model being run
                  x,                         # model input (or a tuple for multiple inputs)
                  "stable_hopenetlite.onnx",   # where to save the model (can be a file or file-like object)
                  export_params=True,        # store the trained parameter weights inside the model file
                  opset_version=10,          # the ONNX version to export the model to
                  do_constant_folding=True,  # whether to execute constant folding for optimization
                  input_names = ['input'],   # the model's input names
                  output_names = ['output'], # the model's output names
                  dynamic_axes={'input' : {0 : 'batch_size'},    # variable lenght axes
                                'output' : {0 : 'batch_size'}})
`

**Comment by ttyio** [comment]
Hello @yueyihua , could you take a try on 8.0EA https://developer.nvidia.com/nvidia-tensorrt-download? thanks

**Comment by ttyio** [comment]
Close since no activity for more than 3 weeks, please reopen if you still have question, thanks!
